### PR TITLE
refactor(service): fixes pixelfed

### PR DESCRIPTION
**Pull request by voryzen**
Pixelfed will deploy, and can be used.

There are some issues that I am aware of; but this merge does not have to wait for them

### CODE REVIEW

File: roles/pixelfed/templates/docker-compose.pixelfed.yml.j2 [hunk 1]
```diff
@@ -1,63 +1,77 @@
+#jinja2: trim_blocks:False
 ---
-version: '3'
```
Comment (codefriar) [inline roles/pixelfed/templates/docker-compose.pixelfed.yml.j2:2]: out of curiosity why are we backing down from version 3 to version 2.x?

**Review by codefriar — APPROVED** [review]

File: roles/pixelfed/templates/docker-compose.pixelfed.yml.j2 (hunk 1 above)
Comment (voryzen) [reply roles/pixelfed/templates/docker-compose.pixelfed.yml.j2:2]: Good question.

Depends_on has been depreciated in version 3, so the conditional health checks will not work

Comment (voryzen) [reply roles/pixelfed/templates/docker-compose.pixelfed.yml.j2:2]: You'll see by this comment from shin- that there is nothing wrong with the version..

https://github.com/docker/compose/issues/4305#issuecomment-305378202

Comment (voryzen) [reply roles/pixelfed/templates/docker-compose.pixelfed.yml.j2:2]: If you are happy with it, please resolve the conversation, and I'll merge

**Pull request merged**